File: app/src/main/java/com/tapan/weather/forecast/core/util/CoreConstants.java
package com.tapan.weather.forecast.core.util;

import java.text.DecimalFormat;

public class CoreConstants {
  public static final String BASE_URL = "https://api.openweathermap.org/data/2.5/";
  public static final String APP_ID = "5ad7218f2e11df834b0eaf3a33a39d2a";
  public static final String DATE_FORMAT = "dd MMM yyyy HH:mm";
  public static final String TABLE_NAME = "GeoLocation";
  public static final String DB_NAME = "GeoLocationDB";
  public static DecimalFormat decimalFormat = new DecimalFormat("#.##");
}

File: app/src/main/java/com/tapan/weather/forecast/core/util/CoreConstants.kt
package com.tapan.weather.forecast.core.util

import java.text.DecimalFormat

object CoreConstants {
  const val BASE_URL = "https://api.openweathermap.org/data/2.5/"
  const val APP_ID = "5ad7218f2e11df834b0eaf3a33a39d2a"
  const val DATE_FORMAT = "dd MMM yyyy HH:mm"
  const val TABLE_NAME = "GeoLocation"
  const val DB_NAME = "GeoLocationDB"
  var decimalFormat = DecimalFormat("#.##")
}